src/sentry/static/sentry/app/views/permissionDenied.tsx

```diff
@@ -1,4 +1,4 @@
-import {withRouter} from 'react-router';
+import {withRouter, WithRouterProps} from 'react-router';
```

Comment:
Is this working without any problems? I know that we are usually importing it like:
```
import {WithRouterProps} from 'react-router/lib/withRouter';
```
or
```
import * as ReactRouter from 'react-router';

... ReactRouter.WithRouterProps ...
```

Reply:
Yeah this works.

I've seen us importing it both ways unfortunately :(